frontend/src/components/SupportGroupChat.jsx

```
@@ -0,0 +1,232 @@
+  const navigate = useNavigate();
+  const [messages, setMessages] = useState([]);
+  const [newMessage, setNewMessage] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [sending, setSending] = useState(false);
+  const [anonymousProfile, setAnonymousProfile] = useState(null);
+  const messagesEndRef = useRef(null);
+
+  useEffect(() => {
+    loadAnonymousProfile();
+    loadMessages();
+  }, [groupId, loadMessages]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)